.teamcity/Gradle_Check/configurations/StagePasses.kt

```diff
@@ -23,7 +23,10 @@ class StagePasses(model: CIBuildModel, stage: Stage, prevStage: Stage?, contains
     artifactRules = "build/build-receipt.properties"
 
     val triggerExcludes = """
-        -:design-docs
+        -:.idea
+        -:.github
+        -:.teamcity
+        -:.teamcityTest
```

Comment:
There are also
- `.gitignore`
- `.editorconfig`
- `.mailmap`
- `.tm_properties`
- `.travis.yml`
- `CONTRIBUTING.md`
- `LICENSE`
- `README.md`
- `gradle.png`

Shall we also ignore the changes to those files?

Reply:
I prefer not. These files are infrequently used, and if we add them here, we probably need to keep `triggerExcludes` up-to-date when we add/remove them.

Reply:
That's fine with me.